packages/wonder-blocks-labeled-field/src/theme/default.ts

```diff
@@ -22,10 +21,6 @@ const theme = {
         },
     },
     description: {
-        font: {
-            size: font.body.size.small,
-            lineHeight: font.body.lineHeight.small,
-        },
```

Comment:
Removing this since the `helperText.font` tokens can be used instead for the `description` so it is shared with other helper text